cli-plugins/socket/socket.go

```diff
@@ -27,6 +27,14 @@ func SetupConn(conn **net.UnixConn) (*net.UnixListener, error) {
 	return listener, nil
 }
 
+func randomID() string {
+	b := make([]byte, 16)
```

Comment:
changed to 16 bytes; which is the same as the UUID (output is slightly shorter as it doesn't add hyphen in between;

```
before: 332becff-7147-44cc-ac72-8ab20eac060e
after:  68298c4c32a19c2b1cc636649c465d84
```